.github/workflows/build.yml

```diff
@@ -42,3 +42,31 @@ jobs:
 
       - name: Run Tests
         run: ./gradlew clean check -x detekt
+
+  pluginVerifier:
+    name: Plugin Verifier
+    needs: gradleValidation
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Fetch Sources
+        uses: actions/checkout@v2
+
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      - name: Setup Gradle Cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
+          restore-keys: ${{ runner.os }}-gradle
+
+      - name: Verify Plugin
+        run: ./gradlew verifyPlugin
```

Comment:
@aldanchenko , we can omit this lines since we already run `verifyPlugin` command in **test** job

Reply:
Good point @pavel-snyk. Thanks!